gradle-baseline-java/src/main/groovy/com/palantir/baseline/tasks/CheckImplicitDependenciesTask.java

```diff
@@ -48,15 +49,19 @@
     private final ListProperty<Configuration> dependenciesConfigurations;
     private final Property<FileCollection> sourceClasses;
     private final SetProperty<String> ignore;
+    private final Property<Boolean> failIfImplicitDependencies;
 
     public CheckImplicitDependenciesTask() {
         setGroup("Verification");
         setDescription("Ensures all dependencies are explicitly declared, not just transitively provided");
-        dependenciesConfigurations = getProject().getObjects().listProperty(Configuration.class);
+        final ObjectFactory factory = getProject().getObjects();
```

Comment:
I think these final keywords are pretty much redundant in modern Java - could we omit them just  to minimise visual clutter?